src/txn/txn_rollback_to_stable.c

```diff
@@ -1216,12 +1303,31 @@ __rollback_to_stable_btree_apply(WT_SESSION_IMPL *session)
                     prepared_updates = true;
             }
             WT_ERR_NOTFOUND_OK(ret, false);
+            ret = __wt_config_subgets(session, &cval, "newest_txn", &value);
+            if (value.len != 0)
+                rollback_txnid = (uint64_t)value.val;
+            WT_ERR_NOTFOUND_OK(ret, false);
             ret = __wt_config_subgets(session, &cval, "addr", &value);
             if (ret == 0)
                 addr_size = value.len;
             WT_ERR_NOTFOUND_OK(ret, false);
         }
         max_durable_ts = WT_MAX(newest_start_durable_ts, newest_stop_durable_ts);
+        has_txn_updates_gt_than_ckpt_snap =
+          WT_CHECK_RECOVERY_FLAG_TXNID_CKPT_SNAPMIN(session, rollback_txnid);
+
+        if (F_ISSET(S2C(session), WT_CONN_RECOVERING))
+            __wt_verbose(session, WT_VERB_RTS,
+              "Recovered checkpoint snapshot min  %" PRIu64 ", snapshot max : %" PRIu64
+              ", snapshot count : %" PRIu32,
+              S2C(session)->recovery_ckpt_snap_min, S2C(session)->recovery_ckpt_snap_max,
+              S2C(session)->recovery_ckpt_snapshot_count);
+
+        /* Increment the inconsistent checkpoint stats counter. */
+        if (has_txn_updates_gt_than_ckpt_snap) {
+            WT_STAT_CONN_INCR(session, txn_rts_inconsistent_ckpt);
```

Comment:
There's a new macro to increment both at once called something like `WT_STAT_CONN_DATA_INCR(session, ...)`.

Reply:
Fixed